File: src/leetcode/com/algorithms/ShuffleTheArray.java
package leetcode.com.algorithms;

public class ShuffleTheArray{

    public static void main(String[] args){

    }

    public static int[] shuffle(int[] nums, int n) {
        int[] ans = new int[2*n];
        int pos =0;
        for(int i =0 ; i < n ; i++){
            ans[pos++] = nums[i];
            ans[pos++] = nums[i+n];
        }

        return int;
    } 
}

File: src/leetcode/com/algorithms/ShuffleTheArray.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function (nums, n) {
  let arr = [];
  for (let i = 0; i < n; i++) {
    arr.push(nums[i], nums[i + n]);
  }
  return ans;
};

shuffle([2, 5, 1, 3, 4, 7], 3);
